gremlin-go/driver/cucumber/cucumberSteps_test.go

```diff
@@ -652,6 +654,22 @@ func compareListEqualsWithoutOrder(expected []interface{}, actual []interface{})
 					break
 				}
 			}
+		} else if reflect.TypeOf(a).String() == "*gremlingo.SimpleSet" {
+			// Set is a special case here because there is no TypeOf().Kind() for sets.
+			actualStringArray := makeSortedStringArrayFromSet(a)
+
+			for i := len(expectedCopy) - 1; i >= 0; i-- {
+				curExpected := expectedCopy[i]
+				if curExpected != nil && reflect.TypeOf(curExpected).String() == "*gremlingo.SimpleSet" {
```

Comment:
nit: better to use type casting with error checking like in https://github.com/apache/tinkerpop/blob/master/gremlin-go/driver/result.go#L153

Reply:
Updated to use type assertion as suggested.